### PR TITLE
JBPM-9044: added needed annotation-api dependency for stunner tests

### PR DESCRIPTION
**Pull request by mareknovotny**
parent PR https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1203

### CODE REVIEW

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/pom.xml [hunk 1]
```diff
@@ -157,6 +157,11 @@
       <artifactId>uberfire-api</artifactId>
     </dependency>
 
+    <dependency>
+        <groupId>org.jboss.spec.javax.annotation</groupId>
```
Comment (yurloc) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/pom.xml:161]: Wrong indentation.

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/pom.xml [hunk 2]
```diff
@@ -157,6 +157,11 @@
       <artifactId>uberfire-api</artifactId>
     </dependency>
 
+    <dependency>
```
Comment (romartin) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/pom.xml:160]: Hey @mareknovotny 

Does it makes sense to make this dependency only available on the `test` scope ( `<scope>test</scope>` ) ? I understand, from the PR's title, that it's only missing on the test classpath. Does that makes sense?

Comment (mareknovotny) [reply kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/pom.xml:160]: @romartin I can check again as I don't recall why i put it in default compile scope

Comment (mareknovotny) [reply kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/pom.xml:160]: done, and still green @romartin

**Pull request merged**